core/tests/protractor_utils/users.js

```diff
@@ -23,11 +23,26 @@ var waitFor = require('./waitFor.js');
 var AdminPage = require('../protractor_utils/AdminPage.js');
 var adminPage = new AdminPage.AdminPage();
 
-var login = async function(email, isSuperAdmin = false) {
+var login = async function(
+    email, isSuperAdmin = false, manualNavigation = true) {
```

Comment:
Could you please add an explanation for the `manualNavigation` arg in some comment? (I understand it but without context, it might be hard to guess why is this exactly needed.)